README.md

```diff
@@ -5,10 +5,14 @@
 
 ## A Specification Framework for the JVM
 
+Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
 
+## Current State of the Project
 
-Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
+- Spek 1.x is in support mode, only bug fixes and documentation updates will be merged into 1.x branch.
+- Spek 2.x is main development branch.
```

Comment:
Are we bringing back `master` at some point?

Reply:
I would say "no" for now, it's just a branch name :) Semantic versioning is easier when branches follow major version so if we would need 3.x that won't require killing master again

Reply:
:+1: